services/state/addressbook/node.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package com.hedera.hapi.node.state.addressbook;
+package proto;
```

Comment:
Please do not move these to the `proto` package.
We will not use `proto` package going forward (we never should have used it at all, as it's an example namespace not to be used in production code).

Reply:
Updated.